playbooks/adhoc/atomic_openshift_tutorial_reset.yml

```diff
@@ -0,0 +1,68 @@
+# This deletes *ALL* Docker images, and uninstalls OpenShift and
+# Atomic Enterprise RPMs.  It is primarily intended for use
+# with the tutorial as well as for developers to reset state.
+
+- hosts:
+    - OSEv3:children
+
+  sudo: yes
+
+  tasks:
+    - service: name={{ item }} state=stopped
+      with_items:
+        - openshift-master
+        - openshift-node
+        - openvswitch
+        - atomic-enterprise-master
+        - atomic-enterprise-node
```

Comment:
The statedir for both is named origin: ```%{_sharedstatedir}/origin``` which is ```/var/lib/origin```

Ref: https://github.com/openshift/origin/pull/4128/files#diff-3477fbcab168ba3ee773ffeb9323e53eR243